connectors/src/connectors/intercom/lib/help_center_permissions.ts

```diff
@@ -381,7 +381,7 @@ export async function retrieveIntercomHelpCentersPermissions({
         expandable: true,
         permission: helpCenter.permission,
         dustDocumentId: null,
-        lastUpdatedAt: null,
+        lastUpdatedAt: helpCenter.updatedAt.getTime(),
```

Comment:
seems unrelated, is it on purpose ?

Reply:
Yes totally unrelated I should have added a comment to make it clear 😬